fix: isPrerendered now set correctly in static configured redirects. (#13924)
withastro/astro: .changeset/sharp-vans-greet.md

---
'astro': patch
---

> Fixed an edge case where `isPrerendered` was incorrectly set to `false` when statically building configured redirects.

Reviewer: ```suggestion
Fixes an edge case where `isPrerendered` was incorrectly set to `false` for static redirects.
```
